doc/guide/getting-started.md

```diff
@@ -172,9 +172,13 @@ Then run this command:
 fusesoc --cores-root=. run --target=synth --setup --build lowrisc:sonata:system
 ```
 
+For the Arty A7 use the `synth_artya7` target.
+
```

Comment:
I think at this early stage it's worth making it clearer why the software image must be built before building the FPGA bitstream, please.

Reply:
I do so a few lines before:
https://github.com/marnovandermaas/sonata-system/blob/1946ed619761084da0168b88ffe0f8ce95f31d13/doc/guide/getting-started.md?plain=1#L169